### PR TITLE
Add routing to the application

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

**Review by DenisKulik — APPROVED** [review]

File: src/routes/router-settings.tsx
Comment (vitalyabedik) [inline src/routes/router-settings.tsx:1]: use camelCase for naming file

File: src/routes/router.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        element: <PrivateRoutes />,
+        children: privateRoutes,
+      },
+      ...publicRoutes,
+      { path: '*', element: <Navigate to={'/404'} /> },
+    ],
+  },
+])
+
+export const Router = () => {
```
Comment (vitalyabedik) [inline src/routes/router.tsx:33]: Maybe do it without return?

File: src/routes/router.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { createBrowserRouter, Navigate, Outlet, RouterProvider } from 'react-router-dom'
+
+import { PrivateRoutes } from './PrivateRoutes'
+import { privateRoutes, publicRoutes } from './router-settings'
+
+import { Header } from '@/components'
+
+// https://stackoverflow.com/questions/75785717/i-am-using-createbrowserrouter-what-is-the-proper-way-to-have-header-and-footer
```
Comment (vitalyabedik) [inline src/routes/router.tsx:8]: we really need it? it usual task

Comment (VitaliKozhurov) [reply src/routes/router.tsx]: Done!

File: src/routes/router.tsx (hunk 1 above)
Comment (VitaliKozhurov) [reply src/routes/router.tsx]: Done!

File: src/routes/router-settings.tsx
Comment (VitaliKozhurov) [reply src/routes/router-settings.tsx:1]: Done!

**Review by vitalyabedik — APPROVED** [review]
Good job!

**Pull request merged**